### DISCUSSION TITLE
Contact

### DISCUSSION THREAD

**Discussion by CycloneRing** [niklas2902/py4godot#46] [General]
Hey, Great work on the project.

I'm a C++ developer but new to Godot, I am working on a project to add a C++based scripting language to godot.
I have some questions since you had experiences on godot api and GDExtension and I'll appreciate if you help me.
It would be nice if you willing to share a email or contact.

Best Regards,
Kenji

**Comment by niklas2902** [comment]
Hey, 

sorry, for getting back so late. Your project sounds very cool. I'm not that big of an expert, but if you have questions, you can message me here

Best Regards,
Niklas
